src/client/routes/Vaults/index.tsx

```diff
@@ -477,6 +481,8 @@ export const Vaults = () => {
                 />
               }
               searching={opportunities.length > filteredVaults.length}
+              filterLabel="Show 0% APY"
```

Comment:
Can you add the locales in the same place as show dust?

Reply:
Seems like we never added the "Show dust" to locales

Reply:
I can add both

Reply:
cool, lets include those then